src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const mateLogo = document.querySelector('.logo');
+
+function createMessage(message, type = '') {
+  const div = document.createElement('div');
+
+  div.innerHTML = `<div class="message ${type}">${message}</div>`;
+  document.body.append(div);
+}
+
+const promise1 = new Promise((resolve) => {
+  mateLogo.addEventListener('click', () => {
+    resolve();
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  setTimeout(() => reject(new Error('Promise was rejected!')), 3000);
```

Comment:
Overall not bad, but with the text it is not what is expected in the task. Doing a little better and more than described in the task is very cool, but not with content. Text and logic must always be reproduced as described in the task. But the fact that you improved the visual is great
```suggestion
  setTimeout(() => reject('Promise was rejected!'), 3000);
```

![image](https://user-images.githubusercontent.com/70009205/223173287-df09128d-474f-4f84-a8d9-adb42efe4996.png)

Reply:
Thanks a lot. I'll do my best )